Validation/Geometry/src/MaterialBudgetData.cc

```diff
@@ -34,14 +35,33 @@ void MaterialBudgetData::SetAllStepsToTree()
   theCoolingDmb     = new float[MAXNUMBERSTEPS];
   theElectronicsDmb = new float[MAXNUMBERSTEPS];
   theOtherDmb       = new float[MAXNUMBERSTEPS];
-  theAirDmb         = new float[MAXNUMBERSTEPS];
+  //HGCal
+  theAirDmb               = new float[MAXNUMBERSTEPS];
```

Comment:
@apsallid - please, consider using std::array or std::tuple. Alternatively, please, make sure the memory is released in the MaterialBudgetData destructor:
```
delete theAirDmb;
```
the same goes to all other pointers.